### PR TITLE
meta: decharter inclusivity wg

### PR DESCRIPTION
**Pull request by jasnell**
Per: https://github.com/nodejs/TSC/issues/218

### CODE REVIEW

File: WORKING_GROUPS.md [hunk 1]
```diff
@@ -24,12 +24,17 @@ Working Group membership or normal TSC motion and vote.
...
-NOTE: Technical Working Groups exist in the [nodejs/node repository](https://github.com/nodejs/node/blob/master/WORKING_GROUPS.md).
+NOTE: Technical Working Groups exist in the [nodejs/ctc repository](https://github.com/nodejs/ctc/blob/master/WORKING_GROUPS.md).
+
+## De-chartered Working Groups
 
 * [Inclusivity](#inclusivity)
 
 ### [Inclusivity](https://github.com/nodejs/inclusivity)
 
+*Note*: The efforts and mission of the Inclusivity Working Group has been
```
Comment (Trott) [inline WORKING_GROUPS.md]: Grammar correction: has -> have   (It's "the efforts and mission" that have been picked up.)

File: WORKING_GROUPS.md [hunk 2]
```diff
@@ -24,12 +24,17 @@ Working Group membership or normal TSC motion and vote.
...
+picked up by the Node.js Foundation's [Community Committee][].
```
Comment (Trott) [inline WORKING_GROUPS.md]: Nit: `have been picked up by` -> `have been assumed by`

**Comment by jasnell** [comment]
Updated!

**Comment by jasnell** [comment]
Given no objections and plenty of time, landing this as approved.

**Pull request merged**